BREAKING.md

```diff
@@ -47,6 +47,7 @@ getBaseGCDetails() has been deprecated in IFluidDataStoreContext and CreateChild
 -   [Remove IConnectableRuntime.deltaManager](#remove-iconnectableruntimedeltamanager)
 -   [IDocumentServiceFactory.protocolName removed](#IDocumentServiceFactory.protocolName-removed)
 -   [Changes to Summarizer's public API](#changes-to-summarizers-public-api)
+-   [`MessageType.RemoteHelp` and the foreman lambda was removed](#messageType-remotehelp-and-the-foreman-lambda-was-removed)
```

Comment:
This should go into the breaking changes section of `2.0.0-internal.5.0.0`. The section already exists in the tip of next, so merging/rebasing should make it show up and we can move the note.